common/_webdriver_qa_api/core/selenium_dynamic_elements.py

```diff
@@ -21,26 +21,28 @@ def name(self):
     def selenium_element(self):
         if self.__parent is None:
             try:
-                logger.info(f"Looking for element {self.__locator}")
+                logger.debug(f"Looking for element {self.__locator}")
                 return self.__driver.find_element(self.__locator_type, self.__locator)
             except NoSuchElementException:
                 logger.exception("An element '{0}' {1}could not be located on the page.".format(
                     self.__name, "" if self.__locator == self.__name else "with locator '{}' ".format(self.__locator)))
+                raise NoSuchElementException
```

Comment:
It affects methods: is_present,  is_present_without_waiting, DropDownMixin.menu_items, scroll_to_element, click_with_js
They are not expected to get an exception here

Reply:
It's ok for DropDownMixin.menu_items, scroll_to_element, click_with_js because we should stop execution if we can't found target element. 

Fixed for is_present